alvr/client_core/src/connection.rs

```diff
@@ -73,8 +73,9 @@ pub struct ConnectionContext {
 
 fn set_hud_message(event_queue: &Mutex<VecDeque<ClientCoreEvent>>, message: &str) {
     let message = format!(
-        "ALVR v{}\nhostname: {}\nIP: {}\n\n{message}",
+        "ALVR v{}\nHMD {}\nhostname: {}\nIP: {}\n\n{message}",
```

Comment:
Could you explain this change seems unnecessary and should be done in a different pr

Reply:
Needed this to check if the model of the HMD was detected correct directly in the app before connecting it to the server. Since I didn't see the model in the log, I thought displaying the name directly in the app would be easier and faster instead of always connecting the client to the server or searching the log

Reply:
Reverted the change